django/http/multipartparser.py

```diff
@@ -656,11 +656,12 @@ def parse_boundary_stream(stream, max_header_size):
     header_end = chunk.find(b"\r\n\r\n")
 
     def _parse_header(line):
-        main_value_pair, params = parse_header(line)
+        main_value_pair, params = parse_header_parameters(line.decode())
```

Comment:
Both `parse_header()` and `_parse_header_params()` are now unused. Please remove them.

Reply:
There is still a subtle use case in http.request.MediaType constructor. I'm going to fix and push it very shortly.